add-ons/dynamic-admission-controllers/dynamic-admission-controllers.addon.remove

```diff
@@ -0,0 +1,18 @@
+# Name: dynamic-admission-controllers
+# Description: Enables ValidatingAdmissionWebhook & MutatingAdmissionWebhook
+# Url: https://docs.openshift.com/container-platform/latest/architecture/additional_concepts/dynamic_admission_controllers.html
+# OpenShift-Version: >=3.10.0
+
+echo Disable required admission configs
+
+ssh sudo echo 's/"admissionConfig": {"pluginConfig": {"ValidatingAdmissionWebhook": {"configuration": {"apiVersion": "v1", "kind": "DefaultAdmissionConfig", "disable": false}}, "MutatingAdmissionWebhook": { "configuration": { "apiVersion": "v1", "kind": "DefaultAdmissionConfig", "disable": false }}}}/"admissionConfig":{"pluginConfig":null}/' > master-config.sed
+ssh sudo sed -i -f master-config.sed /var/lib/minishift/base/kube-apiserver/master-config.yaml
+
```

Comment:
adjust this based on the previous comments ? it will be great if we can roll it back with last known good master-config-yaml.

@praveenkumar @LalatenduMohanty  - thoughts ?